CodeWars/Tasks/ReturnMaximumHourMinute.cs

```diff
@@ -10,26 +10,26 @@ public partial class Tasks
     {
         public static Dictionary<int, int> getFrequencyMap(int[] inputArray)
         {
-            Dictionary<int, int> hashMap = new Dictionary<int, int>();
+            Dictionary<int, int> NumberOfElements = new Dictionary<int, int>();
```

Comment:
https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md

Please, chage acording to Naming Conventions

Reply:
Renamed "NumberOfElements" to "numberOfElements"